<launch>
  <node pkg="voice_common_pkg" name="stt_server" type="stt_server.py" output="screen" />
  <node pkg="gcp_texttospeech" name="tts_srvserver" type="tts_srvserver.py" output="screen" />
  <node pkg="spr_speak" name="spr_speak" type="spr_speak.py"     output="screen" />
</launch>
